Library/Homebrew/utils.rb

```diff
@@ -53,12 +53,6 @@ def truncate(str)
       str.to_s[0, width - 4]
     end
 
-    private
-
-    def color(n)
-      escape "0;#{n}"
-    end
-
```

Comment:
Do we really want to make `Tty.bold` and `Tty.underline` public methods? My feeling is they are okay as private helpers, but too cryptic for public consumption. The only use in this PR seems to be `Tty.underline 39`, but there already exists `Tty.em` for this.